src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -136,11 +136,19 @@ <h5>{% trans "Logic rules problems" %}</h5>
         <ul>
         {% for logic_rule in invalid_logic_rules %}
             <li>
+                {% if logic_rule.exception %}
+                    <p>
+                        {% blocktranslate with index=forloop.counter form_name=logic_rule.form_name trimmed %}
+                            We couldn't process logic rule {{ index }} for '{{ form_name }}' because it appears to be invalid. <br>
```

Comment:
ah - you don't want to use the template for loop counter, but the python code forloop counter! If rule 1 and 2 are fine, but rule 3 has an error, then this would be reported as if it's a problem in rule 1 which is wrong.